test/test-project/gulpfile.js

```diff
@@ -48,5 +68,21 @@ gulp.task('test1', () => {
     // TODO(justinfagnani): allow filtering of files before writing
     .pipe(gulp.dest('build/unbundled'));
 
-  return mergeStream(bundled, unbundled);
+  return Promise.all([waitFor(bundled), waitFor(unbundled)]).then(() => {
+    return Promise.all([
+      addServiceWorker({
```

Comment:
Let's submit as-is, but I think we can get to something like this:

```javascript
return Promise.all([
      addServiceWorker({
        projectConfig: project.unbundledConfig,
      }),
      addServiceWorker({
        projectConfig: project.bundledConfig,
      }),
    ]);
```

Where the project does the work of figuring out which assets comprise the project files, and what the output directory is.